test_util/transaction_test_util.cc

```diff
@@ -350,6 +349,7 @@ Status RandomTransactionInserter::Verify(DB* db, uint16_t num_sets,
             static_cast<int>(key.size()), key.data(), int_value);
         total += int_value;
       }
+      iter->status().PermitUncheckedError();
```

Comment:
Though I agree it is not likely the status is not OK here, you should probably extract the status and return it if it is not OK, rather than just ignoring it.

Reply:
the status must be ok, this is just to work around the status check.